archivy/templates/dataobjs/show.html

```diff
@@ -41,5 +47,64 @@ <h2>{{ dataobj['title'] }}</h2>
 
 			}
 		}
+
+		let editorDiv, oldContent;
+		window.addEventListener("load", function() {
+			let editor = new EasyMDE({
+				autofocus: true,
+				autosave: {
+					enabled: true,
+				},
+				toolbar: [
+					"bold", 
+					"italic", 
+					"strikethrough",
+					"heading",
+					"code",
+					"quote",
+					"link",
+					"image",
+					"side-by-side",
+					"preview",
+					"fullscreen",
+					{
+						name: "save",
+						action: (editor) => {
+							fetch(`${SCRIPT_ROOT}/dataobjs/{{dataobj['id']}}`, {
+								method: "PUT",
+								body: JSON.stringify({
```

Comment:
Doesn't this look weirdly formatted? what is the length of your tab character? For javascript, I guess using `1 tab = 2 spaces` would significantly improve code readability